src/daft-plan/src/logical_ops/explode.rs

```diff
@@ -51,4 +51,18 @@ impl Explode {
             exploded_schema,
         })
     }
+
+    pub fn multiline_display(&self) -> Vec<String> {
+        let mut res = vec![];
+        res.push(format!(
+            "Explode: {}",
+            self.to_explode
+                .iter()
+                .map(|e| e.to_string())
+                .collect::<Vec<_>>()
```

Comment:
with `itertools` you can join without collecting to an intermediate vec
https://docs.rs/itertools/latest/itertools/fn.join.html